presto-orc/src/main/java/com/facebook/presto/orc/metadata/OrcMetadataReader.java

```diff
@@ -149,6 +149,20 @@ private static ColumnEncoding toColumnEncoding(OrcProto.ColumnEncoding columnEnc
         return ImmutableList.copyOf(Iterables.transform(rowIndex.getEntryList(), OrcMetadataReader::toRowGroupIndex));
     }
 
+    @Override
+    public List<HiveBloomFilter> readBloomFilterIndexes(InputStream inputStream)
+            throws IOException
+    {
+        CodedInputStream input = CodedInputStream.newInstance(inputStream);
+        OrcProto.BloomFilterIndex bloomFilter = OrcProto.BloomFilterIndex.parseFrom(input);
+        List<OrcProto.BloomFilter> bloomFilterList = bloomFilter.getBloomFilterList();
+        ImmutableList.Builder<HiveBloomFilter> builder = ImmutableList.<HiveBloomFilter>builder();
```

Comment:
`<HiveBloomFilter>` on the call to `builder()` is not needed.  Your IDE should warn you about this.